### ISSUE TITLE
PhonePrefixSelect breaks if no locale is active

### ISSUE THREAD

**Issue by rfleschenberg** [stefanfoulis/django-phonenumber-field#113]
It is possible that `PhonePrefixSelect` gets instantiated when there is no active locale. For example, i18n is deactivated by default in management commands.

Since Django 1.8, `django.utils.translation.get_language` returns `None` in this case:
https://docs.djangoproject.com/en/1.9/ref/utils/#django.utils.translation.get_language

This will then break on `django.utils.translation.to_locale`. Here is an example traceback:

```
(env)temha:~/development/center$ python manage.py createsuperuser
Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/core/management/__init__.py", line 345, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 348, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/contrib/auth/management/commands/createsuperuser.py", line 52, in execute
    return super(Command, self).execute(*args, **options)
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 398, in execute
    self.check()
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/core/management/base.py", line 426, in check
    include_deployment_checks=include_deployment_checks,
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/core/checks/registry.py", line 75, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/core/checks/urls.py", line 10, in check_url_config
    return check_resolver(resolver)
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/core/checks/urls.py", line 19, in check_resolver
    for pattern in resolver.url_patterns:
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/utils/functional.py", line 33, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/core/urlresolvers.py", line 417, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/utils/functional.py", line 33, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/core/urlresolvers.py", line 410, in urlconf_module
    return import_module(self.urlconf_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/temha/development/center/center/urls.py", line 24, in <module>
    url(r'^accounts/', include('useraccount.urls')),
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/conf/urls/__init__.py", line 52, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/home/temha/development/center/useraccount/urls.py", line 18, in <module>
    from useraccount.views import login, logout, signup
  File "/home/temha/development/center/useraccount/views.py", line 11, in <module>
    from useraccount.forms import *
  File "/home/temha/development/center/useraccount/forms.py", line 39, in <module>
    class UserAccountProfileForm(ModelForm):
  File "/home/temha/development/center/useraccount/forms.py", line 40, in UserAccountProfileForm
    class Meta:
  File "/home/temha/development/center/useraccount/forms.py", line 44, in Meta
    'phone' : PhoneNumberPrefixWidget(),
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/phonenumber_field/widgets.py", line 44, in __init__
    widgets = (PhonePrefixSelect(initial), TextInput(),)
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/phonenumber_field/widgets.py", line 20, in __init__
    locale = Locale(translation.to_locale(translation.get_language()))
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/utils/translation/__init__.py", line 188, in to_locale
    return _trans.to_locale(language)
  File "/home/temha/development/center/env/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py", line 71, in to_locale
    p = language.find('-')
AttributeError: 'NoneType' object has no attribute 'find'
```

Maybe this could be fixed by checking for a `None` return value from `get_language` and falling back to `settings.LANGUAGE_CODE` in this case? This would effectively emulate the behaviour of Django < 1.8.

**Comment by amateja** [comment]
Thank you for reporting. Can you prepare pull request for this issue?

**Comment by rfleschenberg** [comment]
I didn't run into the issue myself, but the person who did is preparing a pull request at the moment :)

**Comment by amateja** [comment]
Fixed with #114.
